defaults_windows_test.go

```diff
@@ -323,7 +328,9 @@ func TestMatchComparerMatch_LCOW(t *testing.T) {
 			match: true,
 		},
 	} {
-		assert.Equal(t, test.match, m.Match(test.platform), "should match %b, %s to %s", test.match, m.Platform, test.platform)
+		if actual := m.Match(test.platform); actual != test.match {
+			t.Errorf("should match %b, %s to %s", test.match, m.Platform, test.platform)
```

Comment:
Heh; looks like the linter now actually picked up an existing bug
<img width="1285" alt="Screenshot 2024-11-19 at 12 19 57" src="https://github.com/user-attachments/assets/66de8197-c61d-4ff0-83af-d01038eccc70">